apidoc/Titanium/UI/iOS/iOS.yml

```diff
@@ -1088,6 +1088,18 @@ properties:
 
   - name: appBadge
     summary: Value of the badge for the application's springboard icon.
+    description: |
+        In order to change the app icon badge number you will need to register for notifications.
+        Normally this is done by using local notifications or a push service like FCM.
+        ``` js
+        Ti.App.iOS.registerUserNotificationSettings({
+          types: [
+                Ti.App.iOS.USER_NOTIFICATION_TYPE_ALERT,
+                Ti.App.iOS.USER_NOTIFICATION_TYPE_SOUND,
+                Ti.App.iOS.USER_NOTIFICATION_TYPE_BADGE
```

Comment:
Only this one is required, the others should be optional. Can you specify this?

Reply:
you are right! I've copied it from another example :)